src/main.rs

```diff
@@ -8,21 +8,21 @@ use axum::{
 };
 use clap::Parser;
 use db_util::{async_db_cleanup, create_db};
-use routes::{
-    browser_html, redirect, redirect_post_multipart_form, return_config_file, return_config_json,
-};
+use routes::{redirect, redirect_post_multipart_form, return_config_file, return_config_json};
 use rusqlite::Connection;
 use std::{fs::create_dir_all, sync::Arc};
 use tokio::sync::Mutex;
-use tower_http::services::ServeFile;
+use tower_http::services::ServeDir;
 
 const DEFAULT_DB_PATH: &str = "/var/lib/wicked_migration_server/db.db3";
-
+const DEFAULT_STATIC_FILE_PATH: &str = "./static";
 #[derive(Parser)]
 #[command(about = "Server to host Wicked config migration", long_about = None)]
 struct Args {
     #[arg(default_value_t = DEFAULT_DB_PATH.to_string())]
     db_path: String,
+    #[arg(default_value_t = DEFAULT_STATIC_FILE_PATH.to_string())]
+    static_files: String,
```

Comment:
I think since there are now more than one argument they should not be purely positional arguments. Because when you want to overwrite just the static files path you also need to overwrite the database path.